.github/workflows/commit-check-pr.yml

```diff
@@ -6,8 +6,6 @@ on:
       - completed
```

Comment:
minor nitpick but `version` and `commit checker` are spelled incorrectly in the commit checker error message: https://github.com/rimolive/data-science-pipelines/pull/2#issuecomment-2142734925

could you add that fix as part of this PR?